samples/GuestConfiguration/installed-application-linux/azurepolicyset.json

```diff
@@ -0,0 +1,27 @@
+{
+    "properties": {
+        "displayName": "Audit if specified applications are not installed inside Linux VMs",
+        "description": "This initiative will both deploy the policy requirements and audit if specified applications are not installed inside Linux virtual machines. For more information on Guest Configuration policies, please visit https://aka.ms/gcpol",
+        "parameters": {
+            "applicationNames": {
+                "type": "String",
+                "metadata": {
+                    "displayName": "Application name"
+                }
+            }
+        },
+        "policyDefinitions": [
+            {
+                "policyDefinitionId": "/providers/Microsoft.Authorization/policyDefinitions/fee5cb2b-9d9b-410e-afe3-2902d90d0004"
+            },
+            {
+                "policyDefinitionId": "/providers/Microsoft.Authorization/policyDefinitions/4d1c04de-2172-403f-901b-90608c35c721",
```

Comment:
We're referencing built-in definitions here, but the sample definitions are different (e.g. applicationNames vs. applicationName parameter)

Either we need the built-in and the sample to be in sync, or this needs to (somehow) reference the custom definitions added here.

Reply:
Agreed.  Per the email with Joel (you were copied), I was just updating the name here, but all these changes/fixes need to be brought back into the built-in.

Reply:
Actually, we can't change the parameter name in the built-in (because it would break existing policy assignments)

So we either shouldn't update it here, or we shouldn't reference the built-in definitions.

Reply:
My concern is that if a customer attempts to copy-paste this, they will get an error about the parameter mismatch.

Reply:
Understood.  Reverted since we are doc'ing the built-in.